src/modules/hmis/components/AssessmentDateWithStatusIndicator.tsx

```diff
@@ -15,7 +15,7 @@ const AssessmentDateWithStatusIndicator = ({
 
   return (
     <Stack direction='row' alignItems='center' gap={0.8}>
-      <Typography variant='inherit' color='error'>
+      <Typography variant='inherit' color='error.dark'>
```

Comment:
Looks like RequiredLabel is another place we should update to `error.dark` instead of just `error`, to avoid contrast issues